### PR TITLE
adding some text

### PR DESCRIPTION
**Pull request by Athulus**
helping with issue #1

### CODE REVIEW

File: domains.txt [hunk 1]
```diff
@@ -105,17 +108,21 @@ javascript
...
 bus drivers
 electricians
 mechanical service
 bus service
 electrical service
+carpentry
 
 # ???
 Civil War reenactment
+Goetia
```
Comment (qguv) [inline domains.txt:137]: VERY GOOD

**Review by qguv — APPROVED** [review]

**Comment by qguv** [comment]
You're the best @Athulus, thanks!

**Comment by Athulus** [comment]
this was fun, you got so many PRs!

**Comment by qguv** [comment]
_fun_

**Pull request merged**